### ISSUE TITLE
In addon/tern, ts.cachedArgHints stores the wrong key, so the cache is ineffective

### ISSUE THREAD

**Issue by dgolds** [codemirror/CodeMirror#3934]
In addon/tern/tern.js, the wrong value is cached for start:

```
 ts.cachedArgHints = {
        start: pos,
```

Instead it should be

```
 ts.cachedArgHints = {
        start: start,
```

This code is towards the end of the   updateArgHints() function

**Comment by marijnh** [comment]
Argh. Thanks for noticing that. Attached patch should improve things.

**Comment by dgolds** [comment]
This doesn't seem to be available yet? the npm package@5.13.4 did not include the fix, so I assume we are still waiting for @5.13.5 to be published?

**Comment by marijnh** [comment]
CodeMirror is released on a time-based schedule, around the 20th of each month, so yes,you'll have to wait a few more days.
